src/transition/progress.js

```diff
@@ -0,0 +1,28 @@
+import {get} from "./schedule.js";
+
+function progressFunction(id, value) {
+  return function() {
+    get(this, id).progress = +value.apply(this, arguments);
+  };
+}
+
+function progressConstant(id, value) {
+  return value = +value, function() {
+    get(this, id).progress = value;
+  };
+}
+
+function abs(value) {
```

Comment:
Math.abs?

Reply:
Yes. I've found the D3 use this instead of `Math.abs`. I think it may be need keep backward compatibility.
So I just copy-paste it.